ap2/utils.py

```diff
@@ -5,6 +5,15 @@
 import subprocess
 
 
+if platform.system() == "Windows":
```

Comment:
In these checks, we only look for Windows. For most cases this will be OK. My concern is just one thing now: whether there are a few people out there who use Windows and pyaudio. Is there a way to restrict this if clause (or put something before it) further by checking whether pyaudio is not in use here, somehow, on Windows? ( Portaudio is multi platform, and if I understand correctly, works on Windows also...)

I know there's nothing there yet for Windows, but... standards. :) 

Perhaps something which checks that the pyaudio device list (on windows) is empty first?

e.g.
```python

p = pyaudio.PyAudio()
info = p.get_host_api_info_by_index(0)
numdevices = info.get('deviceCount')
```

https://people.csail.mit.edu/hubert/pyaudio/docs/

Reply:
Just a thought, but say with `numdevices` we can get at audio startup, so we don't instantiate every time we change the volume.

Reply:
I don't understand your point, what I understand is that pyaudio is a wrapper on top of port audio, so if you're using pyaudio you are using portaudio from python.

Anyway this code has not much to do with either pyaudio or portaudio, windows volume is not linked to the "device" (at least when using pycaw to set the volume) Also what really means device is confusing, e.g. you can have speakers, headphones, HDMI output, etc and those will be devices, I guess pyaudio is just giving us the default device.

Again this PR is using the process audio, it really doesn't matter which device the audio is being played, see images of how windows separate process volumes: https://www.google.com/search?q=windows+volume+mixer&source=lnms&tbm=isch